docs/podman.1.md

```diff
@@ -153,37 +153,41 @@ the exit codes follow the `chroot` standard, see below:
 
-The storage configuration file specifies all of the available container storage options for tools using shared container storage.
+**OCI hooks JSON** (`/etc/containers/oci/hooks.d/*.json`, `/usr/share/containers/oci/hooks.d/*.json`)
 
-When Podman runs in rootless mode, the file `$HOME/.config/containers/storage.conf` is also loaded.
+    Each `*.json` file in `/etc/containers/oci/hooks.d` and `/usr/share/containers/oci/hooks.d` configures a hook for Podman containers, with `/etc/containers/oci/hooks.d` having higher precedence.  For more details on the syntax of the JSON files and the semantics of hook injection, see `oci-hooks(5)`.
 
-**mounts.conf** (`/usr/share/containers/mounts.conf` and optionally `/etc/containers/mounts.conf`)
+    Podman and libpod currently support both the 1.0.0 and 0.1.0 hook schemas, although the 0.1.0 schema is deprecated.
+
+    For the annotation conditions, libpod uses any annotations set in the generated OCI configuration.
 
-The mounts.conf files specify volume mount directories that are automatically mounted inside containers when executing the `podman run` or `podman start` commands. When Podman runs in rootless mode, the file `$HOME/.config/containers/mounts.conf` is also used. Please refer to containers-mounts.conf(5) for further details.
+    For the bind-mount conditions, only mounts explicitly requested by the caller via `--volume` are considered.  Bind mounts that libpod inserts by default (e.g. `/dev/shm`) are not considered.
 
-**hook JSON** (`/usr/share/containers/oci/hooks.d/*.json`)
+    Hooks are not used when running in rootless mode.
 
-Each `*.json` file in `/usr/share/containers/oci/hooks.d` configures a hook for Podman containers.  For more details on the syntax of the JSON files and the semantics of hook injection, see `oci-hooks(5)`.
+**policy.json** (`/etc/containers/policy.json`)
 
-Podman and libpod currently support both the 1.0.0 and 0.1.0 hook schemas, although the 0.1.0 schema is deprecated.
+    Signature verification policy files are used to specify policy, e.g. trusted keys, applicable when deciding whether to accept an image, or individual signatures of that image, as valid.
 
-For the annotation conditions, libpod uses any annotations set in the generated OCI configuration.
+**registries.conf** (`/etc/containers/registries.conf`)
 
-For the bind-mount conditions, only mounts explicitly requested by the caller via `--volume` are considered.  Bind mounts that libpod inserts by default (e.g. `/dev/shm`) are not considered.
+    registries.conf is the configuration file which specifies which container registries should be consulted when completing image names which do not include a registry or domain portion.
 
-Hooks are not used when running in rootless mode.
+    When Podman runs in rootless mode, the file `$HOME/.config/containers/registries.conf` is used.
```

Comment:
On the fence, should we add a "For more details... see containers-registries.conf(5)" line?

Reply:
I think it gets to wordy.  Reason I added this is someone asked me about it.

Actually what they asked about was where for rootless containers the containers are stored.

~/.local/share/containers/storage